src/commonComponents/link/Link.js

```diff
@@ -0,0 +1,39 @@
+// Libraries
+import PropTypes from 'prop-types'
+
+// lodash
+import _noop from 'lodash/noop'
+import _startCase from 'lodash/startCase'
+
+// constats
+import { ACTIVE_CLASS } from './constants/link.general'
+
+
+function Link(props){
+
+    const { category, isActive, onClickHandler } = props;
+
+    function handleClick(){
+        onClickHandler(category);
+    }
+
+    const activeClass = isActive ? ACTIVE_CLASS : undefined;
+
+    return (
+        <p className={activeClass} onClick={handleClick}>{_startCase(category)}</p>
```

Comment:
you are rendering a paragraph element so you should not call this component as Link